app/org/sagebionetworks/bridge/play/controllers/ApplicationController.java

```diff
@@ -18,15 +27,34 @@ public Result loadApp() throws Exception {
         return ok(views.html.index.render());
     }
 
-    public Result verifyEmail(String study) {
-        Study studyObj = studyService.getStudy(study);
+    public Result verifyEmail(String studyId) {
+        if (isBlank(studyId)) {
+            throw new BadRequestException("study parameter is required");
+        }
+        Study study = studyService.getStudy(studyId);
         return ok(views.html.verifyEmail.render(ASSETS_HOST, ASSETS_BUILD,
```

Comment:
Yes, it's for testing. We have a link and it needs to go somewhere, and it might be useful for developers to get a session key. Once installed, an app will intercept this link and process it in the app (using our API to complete the call and capture the session).